src/exec.js

```diff
@@ -21,6 +23,9 @@ function execSync(cmd, opts) {
     silent: common.config.silent
   }, opts);
 
+  var maxBuffer = parseInt(opts.maxBuffer, 10) || DEFAULT_MAXBUFFER_SIZE;
```

Comment:
@pieterjandesmedt I think he meant that `"hello"` is invalid input, so we don't need to support it. I'm fine with either catching the mistake and defaulting to a value (like you've done), or with crashing. I actually prefer your method.